mappings/net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_4790 net/minecraft/world/gen/surfacebuilder/SoulSandValleySurfaceBuilder
+	FIELD field_22204 GRAVEL2 Lnet/minecraft/class_2680;
+	FIELD field_22209 SOUL_SAND2 Lnet/minecraft/class_2680;
```

Comment:
Why `2`?

Reply:
Because GRAVEL and SOUL_SAND exist in the superclass, but :thonkjang: created new fields here

Reply:
Think enigma doesn't allow that. There are public static final fields for gravel and soul sand in the super class `SurfaceBuilder`. You can just change on github and if it's illegal, travis will just fail and notify you.

Reply:
Ah. I think this is that one Enigma bug again (FabricMC/Enigma#191 but for fields this time).

Reply:
Yep, we had trouble for loggers in superclass/child class before.